src/commonMain/kotlin/exchange.dydx.abacus/calculator/TradeInputCalculator.kt

```diff
@@ -1300,7 +1275,6 @@ internal class TradeInputCalculator(
                 account,
                 subaccount,
                 trade,
-                position,
                 market,
```

Comment:
how come `calculatedOptions` does not need `subaccount` as an arg now?`

Reply:
do u mean `position`? If you trace back [here](https://github.com/dydxprotocol/v4-abacus/pull/585/files/4d94186a37f526ea449f53965ffa14a765a024ab#diff-25886307b00a8d85e0a52116d3569ca56fb322bee615ed7731009848b25361c7R168) it's cuz the `maxLeverage` function here (which replaced `maxLeverageFromPosition`) just reads the position from the passed through `subaccount` when it needs it (instead of earlier on). And the reason I passed through subaccount is cuz we now need the equity and freeCollateral off of subaccount as well

Reply:
oh, i missed `subaccount` is already an earlier argument. Nevermind!